build-system/tasks/presubmit-checks.js

```diff
@@ -509,12 +514,6 @@ var forbidden3pTerms = {
   // usage in babel's external helpers that is in a code path that we do
   // not use.
   '\\.then\\((?!callNext)': ThreePTermsMessage,
-  'Math\\.sign': ThreePTermsMessage,
-  'Object\\.assign': {
```

Comment:
I checked the browserified integration.js, which does contain those polylfills already